modules/core/test/test_intrin_utils.hpp

```diff
@@ -2148,7 +2156,6 @@ void test_hal_intrin_float16()
 #endif
 }
 
-
```

Comment:
Why do we have this whitespace change?

It doesn't come from 3.4 patches.

---

These changes are lost in this file: https://github.com/opencv/opencv/blame/fd8b346c3ee36e0ff50acc1cd1bd945b7a77e47b/modules/core/test/test_intrin_utils.hpp#L1948-L1949